src/libraries/System.Drawing.Common/src/misc/GDI/WindowsGraphics.cs

```diff
@@ -110,10 +111,10 @@ public static WindowsGraphics FromGraphics(Graphics g, ApplyGraphicsProperties p
                 }
             }
 
-            if (elements != null)
+            if (offset != default)
```

Comment:
What happens if the offset was retrieved and actually equals default, i.e. 0/0.  Presumably that doesn't matter since TranslateTransform(0, 0) would be a nop?

Reply:
Yes, it would be a no-op.